src/views/file-privacy-masking/ui/Page.tsx

```
@@ -0,0 +1,730 @@
+                        __html: `
+                        @keyframes fly-straight {
+                          0% {
+                            transform: translateX(-100%) rotate(30deg);
+                            opacity: 0;
+                          }
+                          50% {
+                            transform: translateX(0%) rotate(30deg);
+                            opacity: 1;
+                          }
+                          100% {
+                            transform: translateX(120%) rotate(30deg);
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)